examples/grails3-schema-per-tenant/build.gradle

```diff
@@ -23,9 +23,7 @@ dependencies {
     runtimeOnly "org.grails.plugins:scaffolding:$scaffoldingVersion"
 
     testImplementation "org.grails:grails-gorm-testing-support:$testingSupportVersion"
-    testImplementation "org.grails.plugins:geb:$gebPluginVersion", {
-        exclude group: 'org.codehaus.groovy'
-    }
+    testImplementation "org.grails.plugins:geb:$gebPluginVersion"
```

Comment:
Use `org.gebish:geb-spock` instead.

`org.gebish:geb-spock` can be narrowed to `integrationTestImplementation`.

All the selenium modules can be narrowed to `integrationTestRuntimeOnly`.